tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -355,6 +356,44 @@ def unregister_workspace(self, uri: str):
         if uri in self.configurations:
             del self.configurations[uri]
 
+    def _get_enum_inlays(self, decl: NodeAndRange) -> List[InlayHint]:
+        if not self.enum_inlays:
+            return []
+
+        node = decl.node
+        if not isinstance(node, chapel.EnumElement):
+            return []
+
+        enum_value = node.assigned_value()
+        inlay_value: str | None = None
+        if isinstance(enum_value, chapel.IntParam):
+            inlay_value = str(enum_value.value())
+        elif isinstance(enum_value, chapel.UintParam):
+            inlay_value = str(enum_value.value())
+
+        if not inlay_value:
+            return []
+
+        node_init = node.init_expression()
+        edits = []
+        if isinstance(node_init, chapel.Literal):
+            return []
+        elif isinstance(node_init, chapel.AstNode):
+            position = location_to_range(node_init.location()).end
+            value = " /* " + inlay_value + " */"
```

Comment:
consider an edit for this case as well

Reply:
I didn't include it because it's not particularly easy to robustly detect whether such a comment exists. So, you apply the edit, and then the inlay for `/* .. */` is _still_ shown. On the other hand, adding an explicit `= bla` silences the inlay.

Reply:
works for me